docs/source/sections/data.rst

```diff
@@ -4,13 +4,15 @@
 Working with data and querying your results
 *******************************************
 
-In this section of the tutorial, we will focus on how to organise and explore the data in an AiiDA database, based on a previously created database  for this tutorial.
+In this section of the tutorial, we will focus on how to organise and explore the data in an AiiDA database.
+We will use a previously created database for this tutorial.
 To follow the tutorial, you first need to import that archive into your database:
 
 .. code-block:: console
 
    $ verdi import https://object.cscs.ch/v1/AUTH_b1d80408b3d340db9f03d373bbde5c1e/marvel-vms/tutorials/aiida_tutorial_2020_07_perovskites_v0.9.aiida
 
+The import can take several minutes. 
```

Comment:
Yeah, this might be related to the fact that the kubernetes cluster is hosted in Sao Paulo now. Maybe we can already have them start the import before the break?

Reply:
Yes, let's add this action to a list on google docs.